pydicom/valuerep.py

```diff
@@ -43,6 +43,58 @@
 # (the component separator '=' is handled separately)
 PN_DELIMS = {0x5e}
 
+FIXED_VALUE_VRS = ("AS", "DA")
+
+MAX_VALUE_LEN = {
+    "AE": 16,
+    "AS": 4,
+    "CS": 16,
+    "DA": 8,
+    "DS": 16,
+    "DT": 26,
+    "IS": 12,
+    "LO": 64,
```

Comment:
Hmm, its 64 characters maximum, but can't the encoded byte form be longer?

This probably applies to the other unicode VR like PN and LT.

Reply:
That's true - this has to be checked in the encoded form, you are right. Which may be a bit of the problem if we don't know the encoding at that point... Maybe we have to check this while adding it to a dataset, where the encoding is known.

Reply:
It is now checked a second time on encoding for these VRs.